src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -2381,6 +2381,11 @@ internal Binder GetBinder(CSharpSyntaxNode syntaxNode)
                     {
                         diagnostics.Add(ErrorCode.ERR_PartialMethodMustHaveLatent, method.Locations[0], method);
                     }
+
+                    if (method.IsPartialDefinition && (object)method.PartialImplementationPart == null)
+                    {
+                        method.DeclaringCompilation.SymbolDeclaredEvent(method);
```

Comment:
What I mean is I would be more comfortable if instead of moving code from src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs (line 395) to src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs (line 2385) the change simply adjusted condition at the original place.